vuepress/api/README.md

```diff
@@ -374,6 +384,24 @@ Whether suppress warnings outputted when localization fails.
 
 Whether `v-t` directive's element should preserve `textContent` after directive is unbinded.
 
+#### allowHtmlFormatting
+
+> 8.11+
+
+  * **Type:** `AllowHtmlFormattingLevel`
+
+  * **Read/Write**
+
+Whether to allow the use locale messages of HTML formatting.
```

Comment:
🤔 Hmm, I am afirad that I can not understand this description.

Is this feature intended to detect invalid HTML tags in locale message and raise warn/error?
If so, I suggest to name it like `htmlMessageValidationLevel`.

Reply:
Thank you for your reviewing! 

This feature is to detect **HTML tags** from locale messages to provides increased security.
Another alternative, it name like `allowHtmlMessageing`

Reply:
OK, the feature itself sound very nice.

It looks like `off` on `allow～` means allowing HTML without warning.
It is a bit weired for me, since `off` is supposed to disable feature and I thought it means "Do Not Allow".

I suggest

* `allow` instead of `off,
* or, name the configuration `warnHtmlInMessage`

Reply:
Thanks!
>allow instead of `off,
or, name the configuration warnHtmlInMessage

It sounds good.  👍